discord-script.py

```diff
@@ -1,79 +1,68 @@
 import random
+import sys
 import discord
 import scraping_google_images
+import database_creation
 from discord import app_commands
 from discord.ext import commands
-import database-creation
 
 
-class DiscordBot(discord.Client):
-    def __init__(self):
-        super().__init__(intents=discord.Intents.default())
-        self.synced = False
-
-    # bot syncs slash commands on startup
-    async def on_ready(self):
-        await tree.sync(guild=discord.Object(server_id))  # causes slash commands to refresh on bot startup
-        self.synced = True
-        print("Bot is online")
-        database-creation.connect_to_db()
+client = commands.Bot(command_prefix="!", intents=discord.Intents.all())
 
 # Global variables
-bot = DiscordBot()
-tree = app_commands.CommandTree(bot)
 server_id = 1030163730538954853
 
 
-@bot.event
+@client.event
+async def on_ready():
+    print("Bot is online")
+    #database_creation.connect_to_db()

```

Comment:
Why is this commented out?

Reply:
Was commented out because line 19 was breaking the code base. Removing the comment so the database team can work on fixing this issue.